### PR TITLE
Disable codecov status check

### PR DESCRIPTION
**Pull request by ailisp**
coverage is not running now for PRs and only kept for staging/master branch. so we have a code cov number at readme.

Test Plans
----------
This pr doesn't run cov now, can be checked by clicking the gitlab build.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -71,6 +71,9 @@ coverage:
     - ./scripts/build_wasm.sh
     - bash -c 'timeout 60m python3 scripts/parallel_coverage.py'
     - (curl -s https://codecov.io/bash | bash) || true
+    only:
```
Comment (SkidanovAlex) [inline .gitlab-ci.yml]: How does this `only` work?
What PRs will still run coverage?

**Comment by ilblackdragon** [comment]
Should we do this in master? And merge master back to staging?

**Comment by ailisp** [comment]
It’s only apply to master and staging branch. For every pr, it’s mirrored
to gitlab as other branches so coverage would not run.

This should be merge to staging first because majority of prs are based on
staging and will immediately get benefit from this pr

**Comment by ailisp** [comment]
any pr include this version of gitlab ci.yaml it would not run coverage

**Comment by ailisp** [comment]
@ilblackdragon @SkidanovAlex is this okay to merge? their won't be coverage for new PRs once this merged: https://gitlab.com/near-protocol/nearcore/pipelines/115650936

**Comment by bowenwang1996** [comment]
looks like CI failed

**Pull request merged**